code/datums/emergency_calls/emergency_call.dm

```diff
@@ -83,7 +83,19 @@
 	if(SSticker?.mode?.waiting_for_candidates) //It's already been activated
 		return FALSE
 
-	picked_call.mob_max = rand(5, 15)
+	if(name == "Distress Signal") //If we're fighting benos
+		var/xenos = length(GLOB.alive_xeno_list)
+		var/humans = length(GLOB.alive_human_list)
+
+		if(humans / max(1,xenos) > 3) //Humans don't need any further help as they already have advantageous/desirable numbers
+			return FALSE
+
+		var/reinforcements = CLAMP( (xenos - humans * 0.33 ) * 3, 1, 15) //We get the difference between the ideal ratio of 3 humans to 1 xeno, and the current xeno count, then multiply it by 3
```

Comment:
Make this use the config value please instead of your hardcoded ideal 3.

Reply:
Fixed